js/aboutApp/renderer.js

```diff
@@ -9,13 +9,19 @@ renderDom();
 function renderDom() {
     document.addEventListener('DOMContentLoaded', function () {
         const applicationName = remote.app.getName() || 'Symphony';
-        const version = remote.app.getVersion();
         let appName = document.getElementById('app-name');
-        let versionText = document.getElementById('version');
         let copyright = document.getElementById('copyright');
 
         appName.innerHTML = applicationName;
-        versionText.innerHTML = version ? `Version ${version} (${version})` : null;
         copyright.innerHTML = `Copyright &copy; ${new Date().getFullYear()} ${applicationName}`
     });
 }
+
+ipcRenderer.on('buildNumber', (event, buildNumber) => {
+    let versionText = document.getElementById('version');
+    const version = remote.app.getVersion();
+
+    if (versionText) {
+        versionText.innerHTML = version ? `Version ${version} (${version}.${buildNumber})` : 'N/A';
```

Comment:
Does this work for Mac and Windows?

Reply:
This feature is only for windows as we are restricting it in the menu template here: https://github.com/symphonyoss/SymphonyElectron/blob/8fb57d23759da6582c81ef0f3f362fdace185ec9/js/menus/menuTemplate.js#L243